### DISCUSSION TITLE
How to configure completion colors?

### DISCUSSION THREAD

**Discussion by kaathewisegit** [nushell/nushell#12982] [Q&A] [answered]
During tab-completion of paths, the suggestions are highlighted based on their type, similar to the LS_COLORS of `ls`.

Is there a way to configure or disable these colors?

**Comment by fdncred** [comment, marked as answer]
Setting `$env.config.completions.use_ls_colors` to false should disable it. I believe the way to change the colors is to change your LS_COLORS env var.

**Reply by kaathewisegit** [reply]
Thanks!
